### PR TITLE
[tests-only] [full-ci] Added Then steps to improve checks in tests for apiWebdavPreviews suite

### PR DESCRIPTION
**Pull request by grgprarup**
## Description
This PR adds `Then` steps to existing API test scenarios to better check the results of `When` steps.

**Suites covered:**
- `apiWebdavPreviews`

## Related Issue

- Part of https://github.com/owncloud/core/issues/39512

## How Has This Been Tested?

- Locally
- CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

**Review by kiranparajuli589 — COMMENTED** [review]
lgtm :+1:

File: tests/acceptance/features/apiWebdavPreviews/previews.feature [hunk 1]
```diff
@@ -141,9 +141,8 @@ Feature: previews of files downloaded through the webdav API
 
   Scenario: set maximum size of previews
```
Comment (kiranparajuli589) [inline tests/acceptance/features/apiWebdavPreviews/previews.feature:142]: please fix the scenario description while you're here.
```suggestion
  Scenario: preview download with a maximum size of previews set as null
```
or some other nice words

**Comment by kiranparajuli589** [comment]
looks good to me :+1:

File: tests/acceptance/features/apiWebdavPreviews/previews.feature [hunk 2]
```diff
@@ -139,11 +139,10 @@ Feature: previews of files downloaded through the webdav API
     And the value of the item "/d:error/s:exception" in the response about user "Alice" should be "Sabre\DAV\Exception\NotFound"
 
 
-  Scenario: set maximum size of previews
+  Scenario: preview download with a maximum size of previews set as null
```
Comment (individual-it) [inline tests/acceptance/features/apiWebdavPreviews/previews.feature:142]:
```suggestion
  Scenario: download preview of size "null"
```

File: tests/acceptance/features/apiWebdavPreviews/previews.feature [hunk 3]
```diff
@@ -181,16 +180,18 @@ Feature: previews of files downloaded through the webdav API
     Given user "Alice" has uploaded file "filesForUpload/lorem.txt" to "/parent.txt"
     And user "Alice" has downloaded the preview of "/parent.txt" with width "32" and height "32"
     When user "Alice" uploads file with content "this is a file to upload" to "/parent.txt" using the WebDAV API
-    Then as user "Alice" the preview of "/parent.txt" with width "32" and height "32" should have been changed
+    Then the HTTP status code should be "204"
+    And as user "Alice" the preview of "/parent.txt" with width "32" and height "32" should have been changed
 
-  @notToImplementOnOCIS @skipOnOcV10.6 @skipOnOcV10.7 @skipOnOcV10.8.0
+  @skipOnOcV10.6 @skipOnOcV10.7 @skipOnOcV10.8.0
```
Comment (individual-it) [inline tests/acceptance/features/apiWebdavPreviews/previews.feature:186]: does this now work on ocis?

**Review by individual-it — COMMENTED** [review]
looks good, just small comments

File: tests/acceptance/features/apiWebdavPreviews/previews.feature (hunk 3 above)
Comment (grgprarup) [reply tests/acceptance/features/apiWebdavPreviews/previews.feature]: Nope its not working on OCIS. I removed the tag to test on OCIS, but I forgot to put the tag back.

**Review by phil-davis — APPROVED** [review]

**Pull request merged**